FEATURE_MATRIX.md

```diff
@@ -1,93 +1,5 @@
 # Semantic Kernel Feature Matrix by Language
 
-**Legend**
+This document can be found on the Semantic Kernel Documentation site on [Supported Languages.](https://learn.microsoft.com/en-us/semantic-kernel/get-started/supported-languages)
```

Comment:
Can you please update the link on the repo's main README.md too?

Reply:
Done!